pkg/contracts/script/DeployCV.s.sol

```diff
@@ -89,16 +86,26 @@ contract DeployCV is Native, CVStrategyHelpers, Script, SafeSetup {
             StrategyStruct.PointSystem.Unlimited
         );
 
-        uint256 poolIdSignaling = createPool(
+        uint256 poolIdQuadratic = createPool(
             Allo(address(allo)),
             address(strategy2),
             address(registryCommunity),
             registry,
-            address(0),
-            StrategyStruct.ProposalType.Signaling,
-            StrategyStruct.PointSystem.Unlimited
+            address(token),
+            StrategyStruct.ProposalType.Funding,
+            StrategyStruct.PointSystem.Quadratic
         );
```

Comment:
maybe all that changes should be made new file, or with IF to be configurable, because in that way we removed the signaling and others good features for tests, including some address used for other people to test.